services/ymax-planner/src/pending-tx-manager.ts

```diff
@@ -79,14 +86,26 @@ const cctpMonitor: PendingTxMonitor<CctpTx, EvmContext> = {
       ctx.evmProviders[caipId] ||
       Fail`${logPrefix} No EVM provider for chain: ${caipId}`;
 
-    const transferStatus = await watchCctpTransfer({
-      usdcAddress,
-      watchAddress: accountAddress as `0x${string}`,
-      expectedAmount: amount,
-      provider,
-      log: (msg, ...args) => log(`${logPrefix} ${msg}`, ...args),
-      timeoutMs,
-    });
+    let transferStatus = false;
+    if (opts.mode === 'live') {
```

Comment:
Given the binary nature of `opts.mode`, I think a ternary here is worth it to avoid reassignment.
```suggestion
    const watchArgs = {
      usdcAddress,
      toAddress: accountAddress as `0x${string}`,
      expectedAmount: amount,
      provider,
      log: (msg, ...args) => log(logPrefix, msg, ...args),
    };
    const transferStatus = await (opts.mode === 'live'
      ? watchCctpTransfer({ ...watchArgs, timeoutMs: opts.timeoutMs })
      : lookBackCcctp({ ...watchArgs, publishTimeMs: opts.publishTimeMs }));
```